packages/react-codemirror-playground/vite.config.ts

```diff
@@ -3,5 +3,5 @@ import { defineConfig } from 'vite';
 
 export default defineConfig({
   plugins: [react()],
-  base: '/cypher-language-support',
+  base: '',
```

Comment:
Setting this to empty string allowed me to test the built source locally, as well as host it in our demo, see docs: https://vitejs.dev/config/shared-options.html#base